src/main.cpp

```diff
@@ -4338,7 +4338,7 @@ void wait_at_frontend(void)
 {
     struct PlayerInfo *player;
     SYNCDBG(0,"Falling into frontend menu.");
-    NeutralEntranceLevel = 0;
+    creature_entrance_level = 0;
```

Comment:
This setting you added here to the same spot as you also did for the boulder setting. But it seems like a strange place for it. Why not add it at the spot where other game rules are defined?

Reply:
Not per se, but this seems like a strange place to add a variable in the code.

Reply:
Where are similar settings stored? game(add) structure?

Reply:
Adding it to SET_GAME_RULE now would conflict with the boulders direction branch.

Reply:
Having a conflict with a draft pull is not so bad. You can merge the master into that pull after this one is merged to resolve it.
But do you really want this as a 'game rule'. Like my comment above, does it not make more sense like a per-player setting?
And if that's hard/infeasible, it would still make more sense to me to treat it like the 'SET_GENERATE_SPEED' command, which also does not have a random variable in the wait_at_frontend function in main.